pkg/client/log.go

```diff
@@ -4,25 +4,34 @@
 package client
 
 import (
-	"strings"
-
-	"github.com/sirupsen/logrus"
+	"go.uber.org/zap"
 )
 
-func GetLogger(format string, debug bool) *logrus.Logger {
-	logger := logrus.New()
-	logger.SetReportCaller(true)
+func GetLogger(format string, debug bool) (*zap.Logger, error) {
```

Comment:
maybe for a separate PR, but I think we could probably move this logging stuff out of the client pkg and into the main pkg